proteus/mprans/SpatialTools.py

```diff
@@ -836,27 +836,28 @@ class Tank2D(ShapeRANS):
         Domain class instance that hold all the geometrical informations and
         boundary conditions of the shape.
     dim: Optional[array_like]
-        Dimensions of the cuboid.
+        Dimensions of the tank (excluding sponge layers).
     coords: Optional[array_like]
         Coordinates of the centroid of the shape.
     from_0: Optional[bool]
-        If True (default), the tank extends from the origin to postive x, y, z
+        If True (default), the tank extends from the origin to positive x, y, z
     """
     count = 0
 
-    def __init__(self, domain, dim=(0., 0.), coords=None, from_0=True):
+    def __init__(self, domain, dim=(0.,0.), coords=None, from_0=True):
```

Comment:
It could make sense.  We could also add back in `setDimensions` (I had planned to, but delayed it because I wanted to make sure there wouldn't be conflicts - or rather, that those conflicts would bring up immediate errors when they tried to call `setDimensions` wrongly... I guess I forgot to go back and add it in the end) but with its reduced scope of just setting the dimension and maybe calling `_findEdges` (since plain setters are somewhat discouraged/irrelevant in python).